### PR TITLE
Grails 4.0.x

### PR DESCRIPTION
**Pull request by sbglasius**
Upgraded version, verified in local build.

### CODE REVIEW

**Comment by sbglasius** [comment]
@rmorrise  I decided to make my pull-request towards the GPC version (assuming that this is where it will live moving forward). Could you please review. Meanwhile I will try to get the release process up and running with GitHub Actions

File: src/main/groovy/com/cscinfo/platform/constraint/CascadeConstraintRegistration.groovy [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    private static void registerCascadeConstraintOnBeans(ApplicationContext applicationContext,
+                                                         Class interfaceClass,
+                                                         Class clazz,
+                                                         Closure<ConstraintRegistry> closure = Closure.IDENTITY) {
+        Map<String, ?> evaluators = applicationContext.getBeansOfType(interfaceClass)
+        evaluators.each { name, evaluator ->
+            if (clazz.isAssignableFrom(evaluator.getClass())) {
+                ConstraintRegistry reg = closure.call(evaluator)
+                reg?.addConstraint(CascadeConstraint)
```
Comment (rmorrise) [inline src/main/groovy/com/cscinfo/platform/constraint/CascadeConstraintRegistration.groovy:29]: I don't think the spec for this covers the case where reg is null. I'm wondering though, do we really need this check? Or can we throw an exception when it comes up?

Comment (rmorrise) [reply src/main/groovy/com/cscinfo/platform/constraint/CascadeConstraintRegistration.groovy:29]: Or maybe log a warning or something

File: src/main/groovy/com/cscinfo/platform/constraint/CascadeConstraint.groovy [hunk 2]
```diff
@@ -28,6 +30,13 @@ class CascadeConstraint extends AbstractConstraint {
         if (!(constraintParameter instanceof Boolean)) {
             throw new IllegalArgumentException("Parameter for constraint [$CASCADE_CONSTRAINT] of property [$constraintPropertyName] of class [$constraintOwningClass] must be a boolean")
```
Comment (rmorrise) [inline src/main/groovy/com/cscinfo/platform/constraint/CascadeConstraint.groovy:31]: What is the purpose of the validateParameter() method override? Should this check be moved to that method?

File: src/main/groovy/grails/cascade/validation/CascadeValidationGrailsPlugin.groovy [hunk 3]
```diff
@@ -25,37 +26,21 @@ Used with permission.
 '''
     def documentation = "https://github.com/rmorrise/grails-cascade-validation/wiki/How-to-use-cascade-validation"
     def license = "APACHE"
-    def organization = [name: "CSC", url: "http://www.cscglobal.com/"]
+    def organization = [name: "Corporation Service Company", url: "http://www.cscinfo.com/"]
```
Comment (rmorrise) [inline src/main/groovy/grails/cascade/validation/CascadeValidationGrailsPlugin.groovy:29]: This is the company's old name and website; should be CSC at https://www.cscglobal.com

**Review by rmorrise — APPROVED** [review]
Looks good; some nitpicky changes in comments

**Comment by rmorrise** [comment]
Thanks for updating this!

File: src/main/groovy/grails/cascade/validation/CascadeValidationGrailsPlugin.groovy (hunk 3 above)
Comment (sbglasius) [reply src/main/groovy/grails/cascade/validation/CascadeValidationGrailsPlugin.groovy]: I changed it to GPC instead as it will be the new home.

**Comment by sbglasius** [comment]
@rmorrise Looks good now?

File: src/main/groovy/com/cscinfo/platform/constraint/CascadeConstraint.groovy (hunk 2 above)
Comment (sbglasius) [reply src/main/groovy/com/cscinfo/platform/constraint/CascadeConstraint.groovy]: I agree, and I did move it.

**Comment by rmorrise** [comment]
> @rmorrise Looks good now?

Perfect! Thank you!

**Review by rmorrise — APPROVED** [review]

**Pull request merged**